marklogic-data-hub/src/test/java/com/marklogic/hub/deploy/commands/GenerateFunctionMetadataCommandTest.java

```diff
@@ -50,8 +50,8 @@ public void setupEach() throws IOException {
         DocumentWriteSet writeSet = modMgr.newWriteSet();
         StringHandle handle = new StringHandle("'use strict';\n" +
             "\n" +
-            "function testModule(picture, value) {\n" +
-            "  return xdmp.parseDateTime(picture, value);\n" +
+            "function testModule(value, pattern) {\n" +
+            "  return xdmp.parseDateTime(value, pattern);\n" +
```

Comment:
This is a xdmp function, so I think , it has to be changed to xdmp.parseDateTime(pattern, value)